### PR TITLE
Add a sceOpenPSID_driver nid

### PR DESCRIPTION
**Pull request by sum2012**
Used in  "Shin Sangoku Musou 6 special" chinese patched game

### CODE REVIEW

File: Core/HLE/sceOpenPSID.cpp [hunk 1]
```diff
@@ -35,6 +35,21 @@ static int sceOpenPSIDGetOpenPSID(u32 OpenPSIDPtr)
 	return 0;
 }
 
+static int sceOpenPSID_driver_0x19D579F0(u32 OpenPSIDPtr,u32 unknown)
+{
+	ERROR_LOG(HLE, "UNTESTED sceOpenPSID_driver_0x19D579F0(%d,%d)", OpenPSIDPtr,unknown);
+	u8 dummyOpenPSID[16] = { 0x10, 0x02, 0xA3, 0x44, 0x13, 0xF5, 0x93, 0xB0, 0xCC, 0x6E, 0xD1, 0x32, 0x27, 0x85, 0x0F, 0x9D };
```
Comment (hrydgard) [inline Core/HLE/sceOpenPSID.cpp:41]: What is this value?

Comment (hrydgard) [reply Core/HLE/sceOpenPSID.cpp:41]: Looks like OpenPSID is some device-unique value on the PS3? https://www.psdevwiki.com/ps3/OpenPSID  

Is it the same on PSP, or does it even matter? Something used for multiplayer?

Comment (vnctdj) [reply Core/HLE/sceOpenPSID.cpp:41]: Not sure if useful, but, for the record, I've already talked about this variable with @unknownbrackets [a long time ago](https://github.com/hrydgard/ppsspp/issues/4366).

Comment (hrydgard) [reply Core/HLE/sceOpenPSID.cpp:41]: That was indeed a long time ago!

Still curious where these numbers come from, but I'm fine merging a static value here until it's been proven to matter for anything...

Comment (sum2012) [reply Core/HLE/sceOpenPSID.cpp:41]: jpcsp ref 's code
https://github.com/jpcsp/jpcsp/blob/master/src/jpcsp/HLE/modules/sceOpenPSID.java#L45

**Pull request merged**